### ISSUE TITLE
Data not storing in mongodb

### ISSUE THREAD

**Issue by StarAzure** [GavM281/ReactNativePractice#1]
Mongodb Submitting data - but does not write to any collection.

**Comment by GavM281** [comment]
It works for me. Try pull the code again and check again.

Some things you could look at:

- In create.js have you changed the IP address to your own?
- If you start server.js and make requests in something like Postman does it work?
- In server.js have you changed the Mongo URI to your own and does your MongoDB security settings allow access from your IP address?

**Comment by StarAzure** [comment]
I got it to work by installing express and mongoose in the folder.

Question - 
1)I had to start server and the also do expo start. When we deploy this using expo to the appstore, will this be taken care of - starting server and client?
2) I use nextjs and I have code that check if client is already connected to not open new connections. Is this take care of in your example? For example what if 10 users use the app simultaneously and they use
```
app.post('/createUser', (req, res) =>{
app.post('/delete',(req,res) => {
app.post('/update', (req, res) => {
```
3) How to make the post calls more cleaner by putting them in individual api endpoints? 
```
/api/createUser
/api/deleteUser 
...etc
```

4) How to get rid of ip address fetch and make it generic?
